server/opendp_apps/cypress_utils/management/commands/load_demo_data.py

```diff
@@ -0,0 +1,300 @@
+        upload_file.save()
+        self.write_success_msg(f'Data file attached to UploadFileInfo')
+
+        # Add AnalysisPlan
+        #
+        plan = AnalysisPlan(name=f'Plan {get_rand_alphanumeric(7)}',
+                            analyst=analyst_user,
+                            dataset=upload_file,
+                            variable_info=depositor_setup.variable_info,
+                            user_step=AnalysisPlan.AnalystSteps.STEP_0700_VARIABLES_CONFIRMED)
+
+        plan.save()
+        self.write_success_msg(f'AnalysisPlan created: {plan}')
+        self.write_success_msg(f'>> Success! Process complete.', indent=False)
+
+    def is_demo_data_already_loaded(self) -> bool:
+        """Check if any of the demo data is already in the system"""
+        if OpenDPUser.objects.filter(username=self.depositor_username).first() is not None:
+            user_msg = (f'The OpenDPUser "{self.depositor_username}" already exists.'
+                        '\nPlease run "clear_test_data" before attempting to run the "load_demo_data" command')
+            self.stdout.write(self.style.ERROR(user_msg))
+            return True
+
+        if OpenDPUser.objects.filter(username=self.analyst_username).first() is not None:
+            user_msg = (f'The OpenDPUser "{self.analyst_username}" already exists.'
+                        '\nPlease run "clear_test_data" before attempting to run the "load_demo_data" command')
+            self.stdout.write(self.style.ERROR(user_msg))
+            return True
+
+
+        return False
+
+    def write_success_msg(self, user_msg: str, indent=True):
+        """Print output statement"""
+        if indent:
+            user_msg = f'  - {user_msg}'
+        self.stdout.write(self.style.SUCCESS(user_msg))
+
+    def get_depositor_setup_info(self, analyst_user: OpenDPUser) -> DepositorSetupInfo:
+        """Create and return a DepositorSetupInfo object"""
+
+        dataset_questions = {"radio_best_describes": "notHarmButConfidential",
+                             "radio_only_one_individual_per_row": "yes",
+                             "radio_depend_on_private_information": "yes"}
+
+        epsilon_questions = {"secret_sample": "no",
+                             "population_size": "",
+                             "observations_number_can_be_public": "yes"}
+
+        depositor_setup = DepositorSetupInfo.objects.create(
+            creator=analyst_user,
+            user_step=DepositorSetupInfo.DepositorSteps.STEP_0600_EPSILON_SET,
+            dataset_questions=dataset_questions,
+            epsilon_questions=epsilon_questions,
+            variable_info=self.get_variable_info(),
+            default_epsilon=1.0,
+            epsilon=1.0,
+            default_delta=1e-05,
+            delta=1e-05,
+            confidence_level=0.95
+        )
+
+        depositor_setup.save()
+
+        return depositor_setup
+
+    def get_data_profile(self) -> dict:
```

Comment:
Better to make these two either static functions or properties

Reply:
done